SonarQube.TeamBuild.PreProcessor/TargetsInstaller.cs

```diff
@@ -0,0 +1,131 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ITargetsInstaller.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using SonarQube.Common;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace SonarQube.TeamBuild.PreProcessor
+{
+    /// <summary>
+    /// Handlers copying targets to well known locations and warning the user about existing targets file
+    /// </summary>
+    public class TargetsInstaller : ITargetsInstaller
```

Comment:
This class is identical with its equivalent from the bootstrapper, with 2 minor exceptions: 
- the default "installTargets ?" value is now a const  (I used to set it inline)
- the location of the targets is different (it is under "Targets")